api/src/main/java/org/openmrs/module/fhir2/providers/r3/ProcedureRequestFhirResourceProvider.java

```diff
@@ -54,6 +63,34 @@ public ProcedureRequest getProcedureRequestById(@IdParam @NotNull IdType id) {
 		return (ProcedureRequest) VersionConvertor_30_40.convertResource(serviceRequest, false);
 	}
 	
+	@Create
+	public MethodOutcome createProcedureRequest(@ResourceParam ProcedureRequest procedureRequest) {
+		return FhirProviderUtils.buildCreate(
```

Comment:
This needs to be converted back to a `ProcedureRequest` before it's passed to `buildCreate()`